src/k8s_sandbox/_sandbox_environment.py

```diff
@@ -90,7 +91,12 @@ def reorder_default_first(
 
         release = _create_release(task_name, config)
         await HelmReleaseManager.get_instance().install(release)
-        return reorder_default_first(await get_sandboxes(release))
+        sandboxes = reorder_default_first(await get_sandboxes(release))
+
+        if "default" in sandboxes:
+            await sandboxes["default"].run_dropbear()
+
+        return cast(dict[str, SandboxEnvironment], sandboxes)
```

Comment:
If the new typing of `get_sandboxes` is correct, does this need to be cast?

Reply:
Yes, at least according to the typechecker this repo uses. `dict[str, K8sSandboxEnvironment]` isn't a substitute for `dict[str, SandboxEnvironment]` according to the typechecker.